cas-server-webapp/src/main/webapp/WEB-INF/spring-configuration/applicationContext.xml

```diff
@@ -43,17 +45,11 @@
     <aop:aspectj-autoproxy/>
 
     <!--
-      Declare the TimingAspect that we want to weave into the other beans
-      defined in this config file.
-    -->
-    <bean id="timingAspect" class="org.perf4j.log4j.aop.TimingAspect"/>
-
-    <!--
     | The base-package indicates where components stored. Spring will scan this
     | folder and find out the annotated beans and register it in Spring container.
     -->
     <context:component-scan base-package="org.jasig.cas" />
-
+    <mvc:annotation-driven />
```

Comment:
Controllers that deal with statistics and healthcheck require this. This seemed like a good place to add, and is consistent with the way the management app also works and uses this config.